app/templates/orders/view.hbs

```diff
@@ -28,7 +28,8 @@
     </div>
     <div class="six wide column">
       <Orders::EventInfo
-        @data={{this.model.order}} />
+        @data={{this.model.order}}
+        @completed={{true}} />
```

Comment:
Why

Reply:
bcz acc to the enhacement, we want 'go to event page' only for complete order page

Reply:
@iamareebjamal Please let me know

Reply:
You don't need this, you already have order as an argument